extensions/amp-list/0.1/amp-list.js

```diff
@@ -223,35 +197,35 @@ export class AmpList extends AMP.BaseElement {
       this.maybeResizeListToFitItems_();
     });
 
-    const loadMorePromise = this.loadMoreEnabledPromise_.then(enabled => {
-      if (enabled) {
-        this.mutateElement(() => {
-          this.getLoadMoreService_().initializeLoadMore();
-          const overflowElement = this.getOverflowElement();
-          if (overflowElement) {
-            toggle(overflowElement, false);
-          }
-          this.element.warnOnMissingOverflow = false;
-        }).then(() => {
-          this.adjustContainerForLoadMoreButton_();
-        });
-      }
-    });
+    const promises = [];
+
+    if (this.loadMoreEnabled_) {
+      const loadMorePromise = this.mutateElement(() => {
```

Comment:
the old code wasn't really waiting for the `mutatePromise` and I don't think it is needed to be honest anyway. would simplify the code if we don't wait for this

Reply:
Waiting for this has one added benefit of making tests to verify that `layoutCallback` correctly initialised `load-more` elements simpler to write--currently some of the integration tests depend on this. If we want to simplify this, I could refactor this out into a separate function and write a unit test instead.